tools/test_containers_compose

```diff
@@ -2,11 +2,44 @@
 
 set -euo pipefail
 
-for i in webui worker; do
+setup_containers(){
   (
-    cd container/$i/ &&
      for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done &&
     sudo docker-compose up -d &&
     (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
   ) || exit 1;
-done
+}
+
+#test webui
+(
+  cd container/webui/ &&
+  setup_containers &&
+  docker-compose exec -T webui openqa-cli api -X POST jobs ISO=foo.iso DISTRI=my-distri FLAVOR=my-flavor VERSION=42 BUILD=42 TEST=my-test --host http://nginx:9526 || (echo "Error executing a job"; exit 1) &&
+  sleep 1 &&
+  (! docker-compose logs webui | grep "Unable to wakeup scheduler") || (docker-compose logs webui; exit 1)
+) || exit 1;
+
+#test worker
+(
+  cd container/worker/ &&
+  setup_containers
+) || exit 1;
+
+# for i in webui worker; do
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details